functions/bigtable/main_async_test.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import os
+import uuid
+
+from google.cloud import bigtable
+import pytest
+from requests import Request
+
+import main_async
+
+PROJECT = os.environ["GOOGLE_CLOUD_PROJECT"]
+BIGTABLE_INSTANCE = os.environ["BIGTABLE_INSTANCE"]
+TABLE_ID_PREFIX = "mobile-time-series-{}"
+
+
+@pytest.fixture(scope="module", autouse=True)
+def table_id():
+    client = bigtable.Client(project=PROJECT, admin=True)
+    instance = client.instance(BIGTABLE_INSTANCE)
+
+    table_id = TABLE_ID_PREFIX.format(str(uuid.uuid4())[:16])
+    table = instance.table(table_id)
+    if table.exists():
+        table.delete()
+
+    table.create(column_families={"stats_summary": None})
+
+    timestamp = datetime.datetime(2019, 5, 1)
+    rows = [
+        table.direct_row("phone#4c410523#20190501"),
+        table.direct_row("phone#4c410523#20190502"),
+    ]
```

Comment:
Add another entry which has a different prefix, such as `phone$`

Reply:
Similar to above comment, this should stay consistent with the existing sync sample